test/Bugs/inbug.js

```diff
@@ -24,6 +24,14 @@ res &= test1();
 res &= test1();
 res &= test1();
 
+function test2() {
+  var arr0 = Array(1, 2);
+  return (-7 in arr0);

```

Comment:
Add test when a +ve index (> length) is on prototype

Reply:
done! thanks for helping with unittests